egs2/ofuton_p_utagoe_db/svs1/run.sh

```diff
@@ -14,6 +14,7 @@ n_shift=300
 win_length=1200
 
 score_feats_extract=frame_score_feats   # frame_score_feats | syllable_score_feats
```

Comment:
For this specific case, we should use syllable_score_feats instead of frame_score_feats